packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -502,7 +502,7 @@ const NotificationDetail = () => {
               </Stack>
             </Grid>
             <Grid item lg={5} xs={12}>
-              <Box sx={{ backgroundColor: 'white', height: '100%', p: 3 }}>
+              <Box sx={{ backgroundColor: 'white', height: '100%', p: 3, pb: 0 }}>
```

Comment:
Pay attention. Padding must be removed from mobile view. Must be removed also for desktop view?